<!-- -->
<launch>
  	<node pkg="pcl_demo" type="load_pcd_odom" name="load_pcd_odom" output="screen">
		<param name="map_path" value="/home/wlh/map/odom_point_map_with_pole/"/>
		<param name="pose_type" value="/rtk_pose"/>
		<param name="odom_scale" value="100"/>

    </node>

</launch>
